### ISSUE TITLE
Support repeated component registration.

### ISSUE THREAD

**Issue by karger** [zepheira/exhibit3#92]
the registerComponents.exhibit event is triggered to signal components to register themselves.
However, the listeners are all set up to trigger only once, ie $(doc).one("registerComponents.exhibit")
Thus, if a new (second) exhibit is created, it fails to register its components

**Comment by zepheiryan** [comment]
By its original design Exhibit expects to only show one exhibit on a page.  This is not the only obstacle to changing that behavior.  That feature is probably better served by its own pull request / issue.

**Comment by karger** [comment]
Indeed, showing more than one exhibit on a page creates complexity.  However, _reinitializing_ that one exhibit is a natural call for plausible usages of exhibit---for example, exhibit editing.

**Comment by szechyjs** [comment]
Any suggestions on how to reinitialize exhibit? I'm using exhibit in an ember.js app and the entire page doesn't reload on page changes.

simile-widgets/exhibit#124

**Comment by karger** [comment]
Depending on what you need to reinitialize, 
window.exhibit.configureFromDOM() will rerender the exhibit without 
reloading the data.  You can also throw away the whole exhibit and 
recreate the database and load the date afresh; see the autoCreate 
function in exhibit.js in the scripts directory of exhibit.
